client/urls.py

```diff
@@ -20,5 +21,5 @@
     path('help/shopping', views.shopping_help),
     path('help/travel', views.travel_help),
     path('help/workers', views.workers_help),
-    path('volunteer_certificate_image/<int:pk>', views.volunteer_certificate_image_view, name='volunteer_certificate'),
+    # path('download_certificate/<int:pk>', views.download_certificate_view, name='download_certificate'),
```

Comment:
if this view is not required anymore, let's just remove it
```suggestion
```

Reply:
I already did :)